examples/base-path-and-path/tests/base_path_and_path_test.bal

```diff
@@ -13,23 +13,23 @@ function startService() {
     before: "startService",
     after: "stopService"
 }
-function testFunc() {
+function testFunc() returns error? {
     // Invoking the main function.
-    endpoint http:Client httpEndpoint { url: "http://localhost:9090" };
-    // Check whether the server has started. 
+    http:Client httpEndpoint = new("http://localhost:9090");
+    // Check whether the server has started.
     //test:assertTrue(serviceStarted, msg = "Unable to start the service");
     json payload = { "hello": "world" };
     http:Request req = new;
     req.setJsonPayload(payload);
     // Send a `GET` request to the specified endpoint.
     var response = httpEndpoint->post("/foo/bar", req);
```

Comment:
```suggestion
    var response = httpEndpoint->post("/foo/bar", { "hello": "world" });
```
We don't need create a new request here, we can simply pass the payload to the POST action as an argumemt